.gitignore

```diff
@@ -16,4 +16,6 @@ examples/predictors/lstm/models/
 examples/predictors/lstm/predictions/
 examples/predictors/lstm/tests/fixtures/OxCGRT_latest.csv
 
+examples/predictors/tests/fixtures/OxCGRT_latest.csv
```

Comment:
This looks like some local old file path: it doesn't exist anymore. The tests have moved to `examples/predictors/lstm/tests/` as they were specific to the LSTM example.

Reply:
Hmm let me try rerunning to see where it ends up